taxi/urls.py

```diff
@@ -16,11 +22,43 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
-    path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "manufacturers/create/",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
+    ),
+    path(
+        "manufacturers/<int:pk>/update/",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update",
+    ),
+    path(
+        "manufacturers/<int:pk>/delete/",
+        ManufacturerDeleteView.as_view(),
+        name="manufacturer-delete",
+    ),
+    path("cars/",
+         CarListView.as_view(),
+         name="car-list"),
+    path("cars/<int:pk>/",
+         CarDetailView.as_view(),
+         name="car-detail"),
+    path("cars/create/",
+         CarCreateView.as_view(),
+         name="car-create"),
+    path("cars/<int:pk>/update/",
+         CarUpdateView.as_view(),
+         name="car-update"),
```

Comment:
The path name 'car-update' is correct as it uses '-' to split words, which aligns with the checklist requirement.